pkg/yurtadm/cmd/token/token.go

```diff
@@ -50,7 +33,7 @@ func NewCmdToken(in io.Reader, out io.Writer, outErr io.Writer) *cobra.Command {
 		FParseErrWhitelist: cobra.FParseErrWhitelist{UnknownFlags: true},
 		RunE: func(cmd *cobra.Command, args []string) error {
 
-			if err := CheckAndInstallKubeadm(); err != nil {
+			if _, err := exec.LookPath("kubeadm"); err != nil {
 				return err
```

Comment:
how about print the kubeadm download url info to end user?

Reply:
Yes, it will be more user-friendly